### PR TITLE
WIP: Press release review

### PR DESCRIPTION
**Pull request by sebastiankb**
Rendered version: https://cdn.statically.io/gh/w3c/wot-marketing/sebastian-press-release/docs/pressrelease-wot-rec.html?env=dev

>Interoperability of devices and services for the Internet of Things (IoT) with cloud platforms and services 
    at a worldwide scale is a huge challenge.

This sentence appears as the 3rd sentence of the press release and will confuse the reader as it gives the impression that only cloud platforms and services are addressed. We should be more generic here.

### CODE REVIEW

**Comment by sebastiankb** [comment]
>The specifications are based upon rich metadata that describes the data and interaction models exposed to applications, 
  and the communications and security requirements for platforms to communicate effectively.

sounds very abstract and I'm not sure if everybody understand what we try to explain here

File: docs/pressrelease-wot-rec.html [hunk 1]
```diff
@@ -52,8 +52,7 @@ <h2>Web technologies enable the full potential of the Internet of Things by inte
     integration across Internet of Things platforms and
     applications. 
     </p>
-    <p>Interoperability of devices and services for the Internet of Things (IoT) with cloud platforms and services 
-    at a worldwide scale is a huge challenge.<br> 
+    <p><br>
```
Comment (mlagally) [inline docs/pressrelease-wot-rec.html]: Please don't just delete text but rather improve it to address the additional interoperability challenges you are facing.

**Comment by mlagally** [comment]
> > Interoperability of devices and services for the Internet of Things (IoT) with cloud platforms and services
> > at a worldwide scale is a huge challenge.
> 
> This sentence appears as the 3rd sentence of the press release and will confuse the reader as it gives the impression that only cloud platforms and services are addressed. We should be more generic here.

We have to set the scene why we did all the work. If it is only to connect a few devices there are dedicated domain-specific standards, so the added value of using WoT is rather limited.
WoT is most useful if it is widely adopted across manufacturers in large scale scenarios.

File: docs/pressrelease-wot-rec.html (hunk 1 above)
Comment (sebastiankb) [reply docs/pressrelease-wot-rec.html]: done.

**Comment by sebastiankb** [comment]
@mlagally 
>We have to set the scene why we did all the work. 

I provided more generic statement. Please review.

File: docs/pressrelease-wot-rec.html (hunk 1 above)
Comment (sebastiankb) [reply docs/pressrelease-wot-rec.html]: there is an alternative text now. please review.

File: docs/pressrelease-wot-rec.html [hunk 2]
```diff
@@ -52,11 +52,11 @@ <h2>Web technologies enable the full potential of the Internet of Things by inte
...
-    
+    <p><br> 
+   The diversity of technologies in Internet of Things (IoT), such as protocols and data models, typically causes high integration and maintenance 
+	    costs in IoT projects.
   The mission of the <a href="https://www.w3.org/WoT/">Web of Things</a> activities 
-  is to counter the fragmentation of the Internet of Things 
+  is to counter these fragmentation of the IoT
   by forming a Web-based abstraction layer capable of interconnecting existing IoT platforms, devices, 
   and cloud services and complementing available standards. It reduces the
     risk of both investors and customers and enables and encourages
```
Comment (sebastiankb) [inline docs/pressrelease-wot-rec.html]: This sentence included 4 time "and": It reduces the risk of both investors **and** customers **and** enables **and** encourages exponential growth of open markets of devices **and** services.

**Comment by egekorkan** [comment]
I didn't have internet connection to review it on GitHub but I have reviewed it on a pdf that I am attaching to this comment.
[Pressrelease-EGE.pdf](https://github.com/w3c/wot-marketing/files/4297454/Pressrelease-EGE.pdf)

**Comment by sebastiankb** [comment]
@egekorkan many thanks. I will have a look and I will update my PR regarding your input. 
As agreed in the TD call I will add Intel's and IRI testimonial as well. I will then do the merge to have the last common version with all current testimonies and the draft press release, which can be further improved by new iterations.

File: docs/pressrelease-wot-rec.html (hunk 2 above)
Comment (mlagally) [reply docs/pressrelease-wot-rec.html]: >  is to counter these fragmentation of the IoT
Suggest not abbreviate IoT in the first introduction part of the press release.
"these" would be plural, do we want to call out multiple fragmentations?

Comment (sebastiankb) [reply docs/pressrelease-wot-rec.html]: I suggest, this kind of refinement can be decided by the W3C MarkCom team.

Comment (mlagally) [reply docs/pressrelease-wot-rec.html]: I agree that we have many "and"s and the sentences don't flow.

What about:
The  <a href="https://www.w3.org/WoT/">Web of Things</a> counters the fragmentation of the Internet of Things by defining a Web-based abstraction layer for existing platforms, devices, gateways and services.
By complementing existing standards it reduces the risk for investors and customers and enables  rapid growth of open markets for devices and services.

Comment (sebastiankb) [reply docs/pressrelease-wot-rec.html]: much better, we have one "and" less ;-) How about this:

The W3C Web of Things counters the fragmentation of the Internet of Things by defining a Web-based abstraction layer for, e.g., existing platforms, devices, gateways and services. By complementing existing standards, it reduces risk for investors and customers as well as enabling rapid growth of open markets for devices and services.

**Comment by sebastiankb** [comment]
I will merge this PR to get an up-to-date baseline draft in which we can then create new PRs and comments.

**Comment by sebastiankb** [comment]
after the merge I great a new PR which will include the testimonial from Intel and IRI

**Pull request merged**